docs/source/unit-testing.md

```diff
@@ -86,19 +87,7 @@ still possible to use - see their [documentation](pydoc/leapp.html#module-leapp.
 
 #### Testing actors that modify the OS
 
-If your actor is checking the state of the system or modifying it in any way,
-you have two choices to test such functionality:
-- in the test, before executing the actor using the
-`current_actor_context.run()`, change the system on which the tests are executed
-as necessary and after executing the actor, revert back all the changes, so that
-other tests work with the same environment. This option may be quite challenging
-and in some cases not doable, for example when disabling SELinux which requires
-reboot.
-- replace the functions that read or modify the system with functions that do
-not alter the system and return what you specify in the test. This is called
-mocking. Currently it is not possible to mock any function while using the
-`current_actor_context.run()`. But, mocking is possible in an actor's library.
-For that, read further.
+Tests shouldn't modify the system in test.
```

Comment:
I'd leave the sentence that was there in the original form or another, it doesn't matter:
```
- replace the functions that read or modify the system with functions that do
not alter the system and return what you specify in the test. This is called
mocking. Currently it is not possible to mock any function while using the
`current_actor_context.run()`. But, mocking is possible in an actor's library.
For that, read further.
```
The title of this section reads: "_Testing actors that modify the OS_". There are certainly actors that modify the system and this section should tell people how to test such actors.